sway-lsp/src/capabilities/document_symbol.rs

```diff
@@ -53,3 +38,17 @@ pub(crate) fn symbol_kind(symbol_kind: &SymbolKind) -> lsp_types::SymbolKind {
         | SymbolKind::Unknown => lsp_types::SymbolKind::VARIABLE,
     }
 }
+
+#[allow(warnings)]
+// TODO: the "deprecated: None" field is deprecated according to this library
+fn symbol_info(ident: &Ident, token: &Token, url: Url) -> SymbolInformation {
+    let range = get_range_from_span(&ident.span());
+    SymbolInformation {
+        name: ident.as_str().to_string(),
+        kind: symbol_kind(&token.kind),
+        location: Location::new(url, range),
+        tags: None,
+        container_name: None,
+        deprecated: None,
```

Comment:
Could we use `..Default::default()` here and get rid of the warning?

Reply:
Unfortunately `SymbolInformation` doesn't have a impl of `Default` so we need to do this :(